common/app/experiments/Experiments.scala

```diff
@@ -37,15 +26,6 @@ object DarkModeWeb
       participationGroup = Perc0D,
     )
 
-object ServerTracking
-    extends Experiment(
-      name = "server-tracking",
-      description = "Test server test tracking",
-      owners = Seq(Owner.withEmail("fronts.and.curation@guardian.co.uk")),
-      sellByDate = LocalDate.of(2025, 5, 28),
```

Comment:
Can we extend this, and use it to test https://github.com/guardian/fastly-edge-cache/pull/1091 ?

Reply:
Yep, I'll update that